Kconfig.nrf

```diff
@@ -6,6 +6,23 @@
 
 menu "Nordic nRF Connect"
 
+#
+# Provide a new choice to override the mbedtls_external library completely
+# and not have to provide a "dummy" path for the implementation
+#
+choice MBEDTLS_IMPLEMENTATION
+	prompt "Select implementation"
+	default MBEDTLS_LIBRARY_NRF_SECURITY
+
+config MBEDTLS_LIBRARY_NRF_SECURITY
+	bool "Use mbed TLS library from nrf_security"
```

Comment:
minor nit. 
Should this be: `Nordic security backend`.
The corresponding string in nrfxlib here https://github.com/nrfconnect/sdk-nrfxlib/blob/e47d05deb577d45c3c2ea9d53958073f4246123c/nrf_security/Kconfig#L12  is:
`Nordic provided security backend`

I think those prompts should be somewhat aligned. There reader may not know what `nrf_security` is.

Reply:
Fixed.